### PR TITLE
[EDA] Finished picklist task for W-8273355

### PR DESCRIPTION
**Pull request by jofsky**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: cumulusci.yml [hunk 1]
```diff
@@ -21,6 +21,26 @@ sources:
         release: latest
 
 tasks:
+    add_relationship_type_values: 
+        description: Adds Parent's Sibling,Niece,Nephew,and Sibling's Child as picklist values to the Releationship Object's Type picklist field.
```
Comment (jstvz) [inline cumulusci.yml:25]:
```suggestion
        description: "Adds Parent's Sibling, Niece, Nephew, and Sibling's Child as picklist values to the Relationship__c.Type__c field."
```

**Review by jstvz — COMMENTED** [review]
Couple of tweaks suggested.

File: documentation/automation.md [hunk 2]
```diff
@@ -57,6 +57,8 @@
 
 ### Tasks
 
+-   **`add_relationship_type_values`:** Adds Parent's Sibling,Niece,Nephew,and Sibling's Child as picklist values to the Releationship Object's Type picklist field.
```
[the remark this answers is not in the document]
Comment (jofsky) [reply documentation/automation.md:60]: @jstvz I'll be sure to follow this convention moving forward.

File: cumulusci.yml [hunk 3]
```diff
@@ -172,6 +192,7 @@ tasks:
             path: scripts/setup.cls
             apex: qaSetup();
 
+
```
Comment (jstvz) [inline cumulusci.yml:195]:
```suggestion
```

File: cumulusci.yml [hunk 4]
```diff
@@ -440,6 +460,8 @@ flows:
                 task: deploy_trial_config
                 options:
                     unmanaged: False
+            5:
```
Comment (MatthewBlanski) [inline cumulusci.yml:463]: @jofsky Should this be part of the einstein template deploy? It seems like it's not in the right place.

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
That task number is out of order and appears to be in the incorrect location. The task looks great.

File: cumulusci.yml (hunk 4 above)
Comment (jofsky) [reply cumulusci.yml:464]: @MatthewBlanski  I have opened https://github.com/SalesforceFoundation/EDA/pull/1417 to fix everything.

File: cumulusci.yml [hunk 5]
```diff
@@ -126,13 +153,19 @@ tasks:
...
         options:
             path: scripts/setup.cls
             apex: configCourseConnections();
 
     execute_preferred_phone_selection:
         description: Runs anonymous apex to set Preferred Phone to 'Other Phone'.
         class_path: cumulusci.tasks.apex.anon.AnonymousApexTask
+        ui_options:
+            execute_preferred_phone_selection:
+                name: Enable EDA's Preffered Phone Selection Settings
```
Comment (jofsky) [inline cumulusci.yml:168]:
```suggestion
                name: Enable EDA's Preferred Phone Selection Settings
```

File: cumulusci.yml [hunk 6]
```diff
@@ -356,6 +407,63 @@ flows:
...
+                options:
+                    namespaced: True
+            5:
+                task: execute_enable_course_connections
+                options:
+                    namespaced: True
+
+
+    eda_settings_unmanaged:
+        description: Enables various settings on the EDA Settings tab.
```
[the remark this answers is not in the document]
Comment (skosgi1) [reply cumulusci.yml:443]: Just to be explicit.

**Review by skosgi1 — CHANGES_REQUESTED** [review]
@jofsky couple of minor suggestions.

File: cumulusci.yml [hunk 7]
```diff
@@ -356,6 +407,63 @@ flows:
...
+                    namespaced: True
+
+
+    eda_settings_unmanaged:
+        description: Enables various settings in an unmanaged context on the EDA Settings tab.
+        steps:
+            1:
+                task: execute_enable_record_type_validation
+                options:
+                    managed: False
```
Comment (jofsky) [inline cumulusci.yml:448]:
```suggestion
                    namespaced: False
```

**Review by skosgi1 — APPROVED** [review]
Changes look good and happy path testing passes.

**Review by MatthewBlanski — DISMISSED** [review]

**Review by skosgi1 — APPROVED** [review]

**Review by codexmarcel — APPROVED** [review]

**Pull request merged**